atom/browser/atom_browser_client.cc

```diff
@@ -90,14 +98,22 @@ bool AtomBrowserClient::ShouldCreateNewSiteInstance(
     content::BrowserContext* browser_context,
     content::SiteInstance* current_instance,
     const GURL& url) {
-
+  if (g_suppress_renderer_process_restart) {
```

Comment:
Did you mean `g_reuse_renderer_process_for_new_window` ? `g_suppress_renderer_process_restart` is only used during history navigations, I am not sure why it would affect this mode alone.